compiler/one-cmds/one-quantize

```diff
@@ -144,6 +144,12 @@ def _get_parser():
         help=
         "Force MaxPool Op to have the same input/output quantparams. NOTE: This option can degrade accuracy of some models.)"
     )
+    quantization_group.add_argument(
+        '--save_min_max',
+        action='store_true',
+        help=
+        "Save min/max of each tensor. NOTE: Min/max valuse are clipped according to calibration algorithms, such as percentile or moving average. Nudge adjustment is not applied."
```

Comment:
can you plz split this line inside 100 cols?

Reply:
this file has multiple lines that exceed 100 cols.
in another PR, it would be better to update them.

Reply:
Hmm.. It seems that formatting is not applied to one-quantize. Can I merge this as-is and update the format at once?